packages/pwa-kit-runtime/src/utils/ssr-config.server.js

```diff
@@ -10,6 +10,8 @@ const SUPPORTED_FILE_TYPES = ['js', 'yml', 'yaml', 'json']
 
 const IS_REMOTE = Object.prototype.hasOwnProperty.call(process.env, 'AWS_LAMBDA_FUNCTION_NAME')
 
+let targetConfig
```

Comment:
I dunno if this is better?


```suggestion
let memoizedConfig
```

Reply:
or `cachedConfig`